docs/tutorials/index.md

```diff
@@ -31,12 +31,43 @@ Note that it is just a rough classification, and that there are several minor de
 ### Planning simulation
 
 Planning simulation is a simulation type that uses simple dummy data and tests mainly planning/control components.
```

Comment:
I think that if we are going to provide a more detailed explanation in tutorials then we should aim to be precise. When I read that the Planning Simulator `tests mainly planning/control components`, the first thing that comes to mind is "What else does it test?". 

Is it correct to say that the Planning Simulator can also be used to test perception (recognition, tracking and prediction) as well, since it is possible to create dummy objects to see how the ego vehicle responds? 🤔

Reply:
@kenji-miyake @mitsudome-r How about this?

Reply:
> Is it correct to say that the Planning Simulator can also be used to test perception (recognition, tracking and prediction) as well, since it is possible to create dummy objects that the vehicle will respond to?

Actually, no. You can find that planning_simulator does not include perception module,
https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/planning_simulator.launch.xml 
 which means it does not test perception (recognition, tracking and prediction).

Reply:
perception (recognition, tracking and prediction) module can be tested at logging simulator.
I think it is better to be written in the tutorial.

Reply:
> Actually, no. You can find that planning_simulator does not include perception module,
> https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/launch/planning_simulator.launch.xml
> which means it does not test perception (recognition, tracking and prediction).

Okay. So, what is the purpose of the following lines in the planning simulator launch file?

```
  <arg name="perception/enable_detection_failure" default="true" description="enable to simulate detection failure when using dummy perception"/>
  <arg name="perception/enable_object_recognition" default="true" description="enable object recognition when using dummy perception"/>
```

Reply:
If `enable_object_recognition` set true, and when putting dummy objects, tracking and predicting module works.

If `enable_detection_failure` set true, `shape_estimation` will be `/perception/object_recognition/detection/objects`, 
if false, it will be `output/dynamic_object` of `dummy_perception_publisher_node`. 

These are for changing the perception condition for planning.
Also, planning simulator is designed under the assumption that detection, tracking and predicting work correctly